src/sentry/api/endpoints/api_tokens.py

```diff
@@ -45,6 +46,17 @@ def post(self, request):
                 expires_at=None,
             )
 
+            capture_security_activity(
+                account=request.user,
+                type='api-token-generated',
+                actor=request.user,
+                ip_address=request.META['REMOTE_ADDR'],
+                context={
+                    'token': token,
```

Comment:
I don't think we actually need to pass this around if it's not used in the email below, no?